doc/CHANGELOG.md

```diff
@@ -16,10 +16,12 @@ With the portable version, you can easily store it in something like a USB disk,
 ### Fixed
 
 - Fix the wrong default setting for C++ executable file path.
+- Fix Font Size restores to small size after set to default or first launch.
```

Comment:
Are you sure this is a "bug"? The font size definitely changes after setting to default. The bug is #422 and nothing else.

Reply:
I am able to reproduce this in current master.

Reply:
> Are you sure this is a "bug"?

Yes, I tested with my 32-inch monitor, it changes to small size in master.

Here is how to reproduce it:
- Change the font to some other say "Consolas"
- apply the new font
- Hit restore to default and apply settings.
- The new default font has a size less than 12 px, something like 8-9.

Reply:
![image](https://user-images.githubusercontent.com/22212259/85196692-9be07200-b2f9-11ea-89f0-9b18a70cb776.png)